src/app/rundown-view/components/rundown-view/rundown-view.component.html

```diff
@@ -0,0 +1,11 @@
+<sofie-rundown-header *ngIf="rundown" class="c-rundown-header" [rundown]="rundown"></sofie-rundown-header>
+
+<div *ngIf="!rundown; else loadedRundown" class='c-loading-rundown'>
+    <mat-card-title>Loading the rundown...</mat-card-title>
+</div>
+
+<ng-template #loadedRundown>
+    <sofie-rundown [rundown]="rundown"></sofie-rundown>
+</ng-template>
```

Comment:
If you flip the "default state" of this component to be that the Rundown is loaded, you should be able to write something like:
```
<ng-container *ngIf=rundown; else loadingRundown">
  <sofie-rundown-header ...></sofie-rundown-header>
  <sofie-rundown ...></sofie-rundown>
</ng-container>

<ng-template #loadingRundown>
  <mat-card-title>Loading the rundonw...</mat-card-title>
</ng-template>
```

Sidenote: `ng-container` does not get added to the DOM, so it's perfect for elements who has no visiual responsibilities. So you don't need to use a `div` that adds padding and stuff.

Reply:
I really liked that suggestion, it has been implemented.